### DISCUSSION TITLE
Type of data

### DISCUSSION THREAD

**Discussion by nicebread** [RESQUE-Framework/website#103] [Ideas]
add:

- Content data (e.g., social media data, digital traces, media analyses, ...)

**Comment by nicebread** [comment]
Type of Data used (only if empirical and not meta-analyses/ simulations):
- Observational
- Interviews
- Content data (e.g., digital traces, text, audiovisual content)
- Questionnaire
- Behavioral
- Physiological (e.g., fMRI, EEG, peripheral Physiology)
- Other (please specify)

(Vorschlag Medienpsy)

**Comment by nicebread** [comment]
Biopsy:

Physiological (e.g., fMRI, EEG, peripheral physiology) als einzelne Checkboxen (man kann ja alles gemeinsam in einer Studie machen, dann ist das besonders aufwändig)

**Comment by nicebread** [comment]
Make sure that "cross-sectional self-report only" can be applied as an exclusion criterion

**Comment by angaertner** [comment]
-	Use of archival data (e.g., from an organization, health institution)

**Comment by angaertner** [comment]
“Type of Data”: 
Comment: please note that this section confounds type of data and study design; we suggest to distinguish and collect information separately. 

“Type of Data”: please allow for a more detailed description of data that is beyond self-report 
-	As is: “Questionnaire” but specify “collecting self-report data” 
-	Add: Non-self report questionnaire and ratings (e.g., other-ratings such as supervisor ratings, ratings from colleagues, team members, interviewers, etc.)
-	As is: behavioural – but specify “directly measurable data (e.g., reaction time)” 
-	Add: complex behavioural data I - captured with factual information (e.g., fluctuation / turn over / sick leave / paternity leave / length hospitalization) 
-	Add: complex behavioural data II - derived from naturalistic observation or similar (e.g., behaviour in meetings which based on video codings) 
-	As is: physiological
-	Add: Objective data (e.g., sales) and performance data (e.g., theft)
-	Add: Rich / qualitative data and materials (texts, video, audio, interview, big data, social media, … ) used as is or turned into quantitative data 


(Feedback AOW)

**Comment by angaertner** [comment]
Online vs. Verhalten im Labor
